spec/rubocop/cop/lint/deprecated_class_methods_spec.rb

```diff
@@ -3,38 +3,67 @@
 RSpec.describe RuboCop::Cop::Lint::DeprecatedClassMethods do
   subject(:cop) { described_class.new }
 
-  it 'registers an offense for File.exists?' do
-    expect_offense(<<-RUBY.strip_indent)
-      File.exists?(o)
-           ^^^^^^^ `File.exists?` is deprecated in favor of `File.exist?`.
-    RUBY
-  end
+  context 'prefer `File.exist?` over `File.exists?`' do
+    let(:message) { '`File.exists?` is deprecated in favor of `File.exist?`.' }
```

Comment:
We don’t need the `message` `let`s anymore.

Reply:
Ahh. Missed them. Removed now.